### DISCUSSION TITLE
A software can't start after 5.55.15

### DISCUSSION THREAD

**Discussion by dvb001** [sandboxie-plus/Sandboxie#1804] [Q&A Feedback]
Since sandboxie version 5.55.16, 

I can't  run the software "Database .NET" any more
This software website: https://fishcodelib.com/Database.htm
"Database .NET" version: any version 
You can download the newest versoin to test,  Free and Plus Edition 33.9.8121 (2022/03/27)

it's good in sandboxie 5.55.15 or older, please fix it, thanks

**Comment by DavidXanatos** [comment]
There is no 5.50.16 only a 5.55.16 also for me this tool fails also with 5.55.14 please test all the releases and let me know which exactly is the last to work so which exactly broke it.

**Comment by dvb001** [comment]
I can run it with 5.55.15 

![图片](https://user-images.githubusercontent.com/29688940/163674727-17d59877-f900-4883-b049-d6f9e1400d6e.png)

It like this with 5.55.16

![图片](https://user-images.githubusercontent.com/29688940/163674973-36f4207e-e39b-42ab-8382-5c8454432048.png)

My system is Windows 10 21H1 (19043.1645）

Sandboxie Setting


[GlobalSettings]

ActivationPrompt=n
FileRootPath=D:\Sandbox\%USER%\%SANDBOX%
SeparateUserFolders=y
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
EditAdminOnly=n
ForceDisableAdminOnly=n
ForgetPassword=n
NetworkEnableWFP=n
TemplateReject=WindowsRasMan
TemplateReject=WindowsLive
EnableObjectFiltering=y
EnableWin32kHooks=y
Template=RpcPortBindings
Template=7zipShellEx
Template=OfficeLicensing
Template=OfficeClickToRun

[UserSettings_18080305]

SbieCtrl_UserName=zhangys
SbieCtrl_NextUpdateCheck=1650175344
SbieCtrl_UpdateCheckNotify=n
SbieCtrl_ShowWelcome=n
SbieCtrl_EnableLogonStart=n
SbieCtrl_EnableAutoStart=n
SbieCtrl_AddDesktopIcon=n
SbieCtrl_AddQuickLaunchIcon=n
SbieCtrl_AddContextMenu=y
SbieCtrl_AddSendToMenu=n
SbieCtrl_AutoApplySettings=n
SbieCtrl_SettingChangeNotify=n
SbieCtrl_HideWindowNotify=n
SbieCtrl_WindowCoords=1494,859,1078,677
SbieCtrl_ActiveView=40021
SbieCtrl_ShouldDeleteNotify=n
SbieCtrl_ProcessViewColumnWidths=424,70,300
SbieCtrl_ReloadConfNotify=n
SbieCtrl_ResMonNotify=n

[Tools]

Enabled=y
ConfigLevel=9
AutoRecover=y
BlockNetworkFiles=y
RecoverFolder=%Personal%
RecoverFolder=%Favorites%
BorderColor=#00ffff,off,6
ForceFolder=D:\Downloads
CopyLimitKb=4096
CopyLimitSilent=y
BoxNameTitle=n
Template=qWave
PromptForFileMigration=n
OpenPipePath=D:\Downloads
